include/mujoco/mjrender.h

```diff
@@ -39,6 +39,10 @@ typedef enum mjtFramebuffer_ {    // OpenGL framebuffer option
   mjFB_OFFSCREEN                  // offscreen buffer
 } mjtFramebuffer;
 
+typedef enum mjtDepthMapping_ {   // OpenGL depth buffer readout mapping (from znear to zfar)
```

Comment:
I agree because `mjtDepthMap` controls MuJoCo's representation of the depth map regardless of the semantics of the graphics API and buffer.

So I think the comment should be "depth mapping for `mjr_readPixels`".